filebeat/module/traefik/access/test/test.log-expected.json

```diff
@@ -3,53 +3,55 @@
+        "http.request.referrer": "http://example.com/login", 
+        "http.response.status_code": "304", 
+        "http.version": "1.1", 
         "input.type": "log", 
         "log.offset": 0, 
+        "source.ip": "192.168.33.1", 
         "traefik.access.body_sent.bytes": "0", 
-        "traefik.access.http_version": "1.1", 
-        "traefik.access.method": "GET", 
-        "traefik.access.referrer": "http://example.com/login", 
         "traefik.access.remote_ip": "192.168.33.1", 
-        "traefik.access.response_code": "304", 
-        "traefik.access.url": "/ui/favicons/favicon-16x16.png", 
         "traefik.access.user_agent.device": "Other", 
         "traefik.access.user_agent.major": "61", 
         "traefik.access.user_agent.minor": "0", 
         "traefik.access.user_agent.name": "Chrome", 
-        "traefik.access.user_agent.original": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36", 
         "traefik.access.user_agent.os": "Linux", 
         "traefik.access.user_agent.os_name": "Linux", 
         "traefik.access.user_agent.patch": "3163", 
-        "traefik.access.user_name": "-"
+        "url.original": "/ui/favicons/favicon-16x16.png", 
+        "user.name": "-", 
+        "user_agent.original": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36"
     }, 
     {
         "@timestamp": "2017-10-02T20:22:08.000Z", 
         "event.dataset": "access", 
         "event.module": "traefik", 
+        "http.request.method": "GET", 
+        "http.request.referrer": "http://example.com/login", 
+        "http.response.status_code": "304", 
+        "http.version": "1.1", 
         "input.type": "log", 
         "log.offset": 280, 
+        "source.geo.city_name": "Berlin", 
+        "source.geo.continent_name": "Europe", 
+        "source.geo.country_iso_code": "DE", 
+        "source.geo.location.lat": 52.4908, 
+        "source.geo.location.lon": 13.3275, 
+        "source.geo.region_iso_code": "DE-BE", 
+        "source.geo.region_name": "Land Berlin", 
+        "source.ip": "85.181.35.98", 
         "traefik.access.body_sent.bytes": "0", 
-        "traefik.access.geoip.city_name": "Berlin", 
-        "traefik.access.geoip.continent_name": "Europe", 
-        "traefik.access.geoip.country_iso_code": "DE", 
-        "traefik.access.geoip.location.lat": 52.4908, 
-        "traefik.access.geoip.location.lon": 13.3275, 
-        "traefik.access.geoip.region_iso_code": "DE-BE", 
-        "traefik.access.geoip.region_name": "Land Berlin", 
-        "traefik.access.http_version": "1.1", 
-        "traefik.access.method": "GET", 
-        "traefik.access.referrer": "http://example.com/login", 
         "traefik.access.remote_ip": "85.181.35.98", 
-        "traefik.access.response_code": "304", 
-        "traefik.access.url": "/ui/favicons/favicon.ico", 
         "traefik.access.user_agent.device": "Other", 
```

Comment:
We have to `user_agent` in ECS? Feels like something fairly important?

@ruflin @webmat A more general comment: For the fields that are not in ECS, but are still present in multiple modules (apache, nginx, traefik, iis, etc.), wouldn't it be better to place them under `http.request`/`http.response` rather than technology-specific fields? That would allow them to be used in "ECS dashboards", otherwise it doesn't work.

Reply:
@webmat Let's sync up on the user_agent issue, not sure I understand the problem.

Reply:
I don't either, don't worry 😂